### PR TITLE
Boost new Pokemon egg steps

### PR DESCRIPTION
**Pull request by Qwertypop04**
Boosts the egg steps of Pinkan Arbok, Poliwhirl and Weezing, and Aegislash (Blade) to match their base forms

### CODE REVIEW

**Comment by RedSparr0w** [comment]
For the evolutions, can you please just put them as unobtainable evolution methods, so we don't need to manually change the egg steps if it's adjusted down the road.

**Comment by Qwertypop04** [comment]
Okay. Although that'll look really weird in bot coins lol

**Comment by RedSparr0w** [comment]
I think we need to just eventually add a `evolutionStage` argument or similar so the egg steps get calculated off that when it doesn't have actual evolution stages.

**Comment by Qwertypop04** [comment]
Yeah. That would also solve the wierd cases like burmy, where a Pokémon evolves from another form of the same Pokémon

**Comment by Jaaslet** [comment]
@RedSparr0w, it seems like this solution will do so the bot will give a lot of false information. Don't know if you just want to make an exception for the bot or something?
But if the solution is fine, you can just approve it and merge it. I have already tested it, and it looks good!

File: src/scripts/pokemons/PokemonList.ts [hunk 1]
```diff
@@ -657,7 +662,7 @@ const pokemonList = createPokemonArray(
     {
         'id': 12.03,
         'name': 'Ash\'s Butterfree',
-        'catchRate': 45,
+        'catchRate': 15,
```
Comment (RedSparr0w) [inline src/scripts/pokemons/PokemonList.ts:665]: Was this intentional?

File: src/scripts/pokemons/PokemonList.ts [hunk 2]
```diff
@@ -1911,11 +1927,11 @@ const pokemonList = createPokemonArray(
     {
         'id': 45.02,
         'name': 'Pinkan Vileplume',
-        'catchRate': 45,
+        'catchRate': 20,
```
Comment (RedSparr0w) [inline src/scripts/pokemons/PokemonList.ts:1930]: Was this intentional?

**Review by RedSparr0w — APPROVED** [review]
Looks good, just a couple of places where values changed, that I'm not too sure were intentional 😅 

> Don't know if you just want to make an exception for the bot or something?

Will work something out, for now will probably just ignore the words `Pinkan` and `Valencian`

File: src/scripts/pokemons/PokemonList.ts [hunk 3]
```diff
@@ -640,11 +645,11 @@ const pokemonList = createPokemonArray(
     {
         'id': 12.02,
         'name': 'Pink Butterfree',
-        'catchRate': 45,
+        'catchRate': 15,
```
[the remark this answers is not in the document]
Comment (Qwertypop04) [reply src/scripts/pokemons/PokemonList.ts]: No.

File: src/scripts/pokemons/PokemonList.ts (hunk 1 above)
Comment (Qwertypop04) [reply src/scripts/pokemons/PokemonList.ts]: No.

File: src/scripts/pokemons/PokemonList.ts (hunk 2 above)
Comment (Qwertypop04) [reply src/scripts/pokemons/PokemonList.ts]: No. I have undone all 3 of those mistakes

**Pull request merged**